app/router/router.tsx

```diff
@@ -100,6 +100,16 @@ class Router {
     this.navigateTo(Path.commitHistoryPath + commit);
   }
 
+  navigateToCreateOrg() {
+    if (!capabilities.createOrg) {
+      alert(
```

Comment:
what if we do this here:
```
window.open('https://buildbuddy.typeform.com/to/PFjD5A', "_blank") 
```

then the call site can be super clean with no need for preventDefault, path, <a href=, target, importing Path, etc

just a ```<div className="clickable" onClick={() => router. navigateToCreateOrg()}>Create organization</div>```

Reply:
Done

Reply:
FYI, the `.clickable` `div` is not keyboard accessible (unlike `<a>`) -- we might want to fix that in a clean sweep in the future.